wellcad/com/_borehole.py

```diff
@@ -2351,6 +2445,38 @@ def calculate_casing_thickness(self, log=None, prompt_user=None, config=None):
         """
         self._dispatch.CalculateCasingThickness(log, prompt_user, config)
 
+    def cased_hole_ultrasonics(self, abiwavelet_log=None, zone_log=None, prompt_user=None, config=None):
```

Comment:
Probably best to just call this `wavelet_log`, given it isn't exclusive to the ABI. If you want to keep the term ABI, go with `abi_wavelet_log`.